src/api-reference/budget/budget-trackingfield.md

```diff
@@ -1,122 +1,172 @@
 ---
-title: Budget Tracking Field
+title: Budget Tracking
 layout: reference
 ---
 
-PRE-RELEASE: SAP Concur’s Budget API is currently in PRE-RELEASE and is subject to change. An Early Adopter program is available. To become an early adopter, contact your SAP Concur Representative and have them contact Team Adaptation.
+## Menu
 
-# Budget Tracking Field
+* [Getting Started](./getting-started.html)
+* [Fiscal Year](/api-reference/budget/fiscal-year.html)
+* [Budget Category](/api-reference/budget/budget-category.html)
+* [Budget Item](/api-reference/budget/budget-header.html)
+* [Budget Tracking Field](/api-reference/budget/budget-tracking.html)
+* [Budget Adjustments](/api-reference/budget/budget-adjustments.html)
 
-- [Retrieve all Budget Tracking Field](#getall)
-- [Retrieve a Budget Tracking Field](#get)
-- [Create/Update a Budget Tracking Field](#post)
-- [Remove a Budget Tracking Field](#delete)
-- [Schema](#schema)
+**Preview** _This is a prerelease version of the service and is subject to change before final release._
 
-## Version
-
-4.0
+This resource is used to retrieve information about budget's tracking fields for an entity. Every entity may have a specific set of budget tracking fields and every budget may enable any or all of the budget tracking fields. If there are tracking fields associated, the  budgets get matched to the Product only when the tracking field conditions are met. 
```

Comment:
Noticed these:

```
If there are tracking fields associated, the  budgets... <-- Has two spaces mid-sentence.
Retrieve budget tracking fields for use in budget configuration.  The budget sequence... <-- Has two spaces after a sentence.
```

Some stylesheets make these more obvious than others, so this is a bit of future proofing -- can we go ahead and scrub and ensure single space? Thanks.

Reply:
:+1: